### DISCUSSION TITLE
Issue: PNG and Colors Not Displaying on Startup in Kitty Terminal

### DISCUSSION THREAD

**Discussion by nsk-reddy** [fastfetch-cli/fastfetch#1437] [Q&A] [answered]
Hey everyone,

I’m new to terminal customization and trying to set up fastfetch in my terminal (Kitty) using Zsh. I added it to my .zshrc, so it runs automatically, but I’m having two issues:

- The PNG image doesn’t load automatically when I open a terminal.

- The colors also don’t display properly on startup.

When I manually run fastfetch, everything works perfectly—the image and colors show up fine.

Here’s what I’ve tried:

- Adding a delay in .zshrc like (sleep 0.5 && fastfetch) &.

- Confirmed $DISPLAY is set (I’m using Wayland with Hyprland).

Manually tested fastfetch,and it works perfectly when done manually.

Here’s what it looks like when it works:
![image](https://github.com/user-attachments/assets/8fbe191b-eb98-4245-8bd6-bce9c6cafc1d)

Any idea how to fix this? Would really appreciate the help! 😊

Thanks in advance!

**Comment by CarterLi** [comment, marked as answer]
Please check this first: https://github.com/fastfetch-cli/fastfetch?tab=readme-ov-file#q-fastfetch-runs-in-white-and-black-on-shell-startup-why 

If it doesn't work for you, you should file a bug and fill the template carefully. Thanks.

**Reply by nsk-reddy** [reply]
Thanks a lot for your help! Putting Fastfetch before the p10k instant prompt in my .zshrc fixed the issue, and everything’s working perfectly now. I really appreciate your quick response!

**Reply by adiXjinx** [reply]
it worked for me too, thanks !
